services/alerta/service.go

```diff
@@ -104,12 +116,12 @@ func (s *Service) Alert(token, resource, event, environment, severity, group, va
 		return errors.New("Resource and Event are required to send an alert")
 	}
 
-	url, post, err := s.preparePost(token, resource, event, environment, severity, group, value, message, origin, service, data)
+	req, err := s.preparePost(token, resource, event, environment, severity, group, value, message, origin, service, data)
 	if err != nil {
 		return err
 	}
 
-	resp, err := http.Post(url, "application/json", post)
+	resp, err := s.client.Do(req)
```

Comment:
Lastly this becomes:

```
client := s.clientValue.Get().(*http.Client)
resp, err := client.Do(req)
```

All of these changes avoid a race condition on the `client` field, synchronizing access to the field.